### ISSUE TITLE
Number of threads inconsistent?

### ISSUE THREAD

**Issue by KatharinaHoff** [gatech-genemark/ProtHint#3]
Hi Tomas,

I call ProtHint as follows:

```
prothint.py --threads=10 --geneMarkGtf  genemark.gtf genome.fa proteins.fasta
```

The log says:

```
...
#CPU threads: 12
Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
Database file: /nas-hs/projs/braker/topaz/test_prothint/fly/proteins.fasta
Opening the database file...  [0.330323s]
...
#CPU threads: 10
Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
Temporary directory: 
Opening the database...  [0.003124s]
#Target sequences to report alignments for: 250
...
```

I kind of expected the pipeline to never use more than 10 threads when specifying `--threads=10`.

Might look like a lot of issues, today, but don't worry, overall, ProtHint is running fine on my machine.

Best,

Katharina

**Comment by tomasbruna** [comment]
Hi Katharina,

thanks for spotting this! When building DIAMOND database, a parameter specifying the number of threads is missing and all threads are used in that step. I will fix this.

Tomas

**Comment by tomasbruna** [comment]
Fixed in 221767047d3964cdb9b9e0d0df808b415e2fb3da
